plugins/paymethod/manual/mailables/ManualPaymentNotify.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @file classes/mail/mailables/ManualPaymentNotify.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ManualPaymentNotify
+ * @ingroup mail_mailables
+ *
+ * @brief Email is sent automatically to notify journal manager about new payment that needs to be processed
+ */
+
+namespace APP\plugins\paymethod\manual\mailables;
+
+use APP\core\Application;
+use APP\journal\Journal;
+use PKP\mail\Mailable;
+use PKP\mail\traits\Configurable;
+use PKP\mail\traits\Sender;
+use PKP\payment\QueuedPayment;
+use PKP\security\Role;
+use PKP\user\User;
+
+class ManualPaymentNotify extends Mailable
+{
+    use Configurable;
+    use Sender;
+
+    protected static ?string $name = 'plugins.paymethod.manual.manualPaymentNotify.name';
+    protected static ?string $description = 'emails.manualPaymentNotification.description';
+    protected static ?string $emailTemplateKey = 'MANUAL_PAYMENT_NOTIFICATION';
+    protected static array $toRoleIds = [Role::ROLE_ID_MANAGER];
+
+    protected static string $paymentName = 'paymentName';
+    protected static string $paymentAmount = 'itemCost';
+    protected static string $paymentCurrencyCode = 'itemCurrencyCode';
+    protected static string $senderUsername = 'senderUsername';
```

Comment:
I think we've had this discussion before but can we use protected constants?

```php
protected const PAYMENT_NAME = 'paymentName';
```

Reply:
OK. Completely forgot here that there is `QueuedPaymentEmailVariable` object in the lib core.